unlearnfeature.go

```diff
@@ -89,6 +90,17 @@ func (f *UnlearnFeature) Execute(s DiscordSession, channel string, command *Comm
 	if command.Unlearn == nil {
 		fatal("Incorrectly generated unlearn command", errors.New("wat"))
 	}
+
+	// Get the current channel and check if we're being asked to unlearn in a
+	// private message.
+	c, err := s.Channel(channel)
```

Comment:
'channel' is already in use - happy to use something else, but one character var names for things like this are idiomatic go IM(admittedly uneducated)O